casbin/rbac/default_role_manager/role_manager.py

```diff
@@ -47,9 +47,11 @@ def clear(self):
 
     def add_link(self, name1, name2, *domain):
```

Comment:
Why is `domain` declared as variable list, when it should only be one parameter?
I think it's best to change `*domain` to `domain=None`. 
[`get_roles`](https://github.com/casbin/pycasbin/blob/3ca1d8e2c6e4ef9623ce815323576b37551e41a7/casbin/rbac/default_role_manager/role_manager.py#L112) can be used as reference.

Reply:
@abichinger I have thought of doing this, but it is more flexible to use `domain` as a variable parameter, because `RoleManager` can be implemented in multiple ways .